common/static/common/js/spec_helpers/ajax_helpers.js

```diff
@@ -18,12 +18,13 @@ define(['sinon', 'underscore'], function(sinon, _) {
      * Get a reference to the mocked server, and respond
      * to all requests with the specified statusCode.
      */
-    fakeServer = function (statusCode, that) {
+    fakeServer = function (that, response) {
         var server = sinon.fakeServer.create();
         that.after(function() {
             server.restore();
         });
-        server.respondWith([statusCode, {}, '']);
+        server.respondWith(response);
+        server.autoRespond = true;
```

Comment:
The sinon docs [say](http://sinonjs.org/docs/#responses):

> Boolean request.autoRespond
> When set to true, causes the server to automatically respond to incoming requests after a timeout. The default timeout is 10ms but you can control it through the autoRespondAfter property. Note that this feature is intended to help during mockup development, and is not suitable for use in tests.

If we need to set autoRespond to true, can you please add a comment as to why?